source/_patterns/basics/dropdowns/00-kebab.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Kebab
+---
+## Overview
+
+The kebab is a variation of a dropdown where the dropdown button displays an ellipsis icon instead of text.
+
+## Usage
+
+| Class | Usage |
+| -- | -- |
+| `.pull-right` **Applied to:** `.dropdown` |  **Outcome:** Floats the kebab to the right of the siblings that follow it.  **Required:** No  |
```

Comment:
`pull-right` is a utility class that can be added to any element to float it. I don't think this should be described here as usage of this particular element.